edx_proctoring/api.py

```diff
@@ -453,6 +467,14 @@ def get_exam_attempt_by_id(attempt_id):
     return _get_exam_attempt(exam_attempt_obj)
 
 
+def get_exam_attempt_by_external_id(external_id):
+    """
+    Return an existing exam attempt for the given student
```

Comment:
There's no student given.

It would be great if we could update the docstrings to be clear about what is happening. "Attempt" is used to mean (I think) three different things: our attempt id, their attempt id, and our attempt object.